vpc/proto/vpc.proto

```diff
@@ -262,6 +277,10 @@ service TitusAgentVPCService {
     rpc GCV2 (GCRequestV2) returns (GCResponseV2);
     rpc GCV3 (GCRequestV3) returns (GCResponseV3);
 
+    // Lock Management
```

Comment:
In general the pattern has been something like:

```
message GetLocksRequest {
}
```

So, if we wanted to add filtering (one day?) we could.

Reply:
Yep, good call. I was following the style of titus-api-definitions but I absolutely see your point.

Reply:
Hm, looking back at the titus-api-definitions I don't see what I was thinking. Anyway, I'll get that change in.